subcommands/targets/offline-update.go

```diff
@@ -62,6 +63,8 @@ func init() {
 		"Fetch only TUF metadata")
 	offlineUpdateCmd.Flags().BoolVarP(&ouNoApps, "no-apps", "", false,
 		"Skip fetching Target Apps")
+	offlineUpdateCmd.Flags().BoolVarP(&ouForce, "force", "", false,
+		"Cleanup previous downloaded update data from <dst>, if not empty")
```

Comment:
I'm on the fence here.

This looks about right; but we use `--force` argument when we want to overcome the server-side validation.
Here, what you do is overwrite the local files.

Just IMHO, the user should do the directory cleanup on their own.
But, if you do want a flag, I'd call it smth like `--overwrite`.

Reply:
This is the request from customer support.

I am fine with either `--force` or `--overwrite`. Another option is `--cleanup`.

Reply:
Thing about a usability feature like this: The user is not going to know about the flag, and hits a "this exists error" when they try the command. Either way, they have to run 2 commands.

Seems like the real usability fix would be to auto-delete the directory for the user. But I think we all agree that's dangerous (say they made did `fioctl targets ... $HOME`). I'm not sure I want our code to delete anything - I'd rather make them run `rm -rf $HOME` than `fioct targets .... --force $HOME`

Reply:
User might not know what exactly to remove inside the dst folder.

Having said that, this approach contradicts with another request - support multiple targets in one update source directory.

Reply:
Maybe, the command should fail if these two folders exists by default.

Then add another option `--allow-multiple-targets` which would allow customers to pull another Target data to the dst folder.

Reply:
So, I see us mixing many things here:
- 1. A safety check that a user does not delete important data.
- 2. An usability feature to cleanup that date _on behalf_ of the user.
- 3. Another usability feature to allow adding more content to existing folder.

Just IMHO, it is better to implement 1 without 2 in any case.
Furthermore, 2 and 3 kind of contradict each other.

That's why I like the latest suggestion from @mike-sul about `--allow-multiple-targets` to implement 3.

Reply:
OK, I think we should not remove any folder too. Let's just check and fail if one or two of those folders (ostree_repo and apps) exists, unless `--allow-multiple-targets` is explicitly specified.